modules/network/src/main/java/org/apache/ignite/network/scalecube/ScaleCubeTopologyService.java

```diff
@@ -104,6 +115,18 @@ void onMembershipEvent(MembershipEvent event) {
         }
     }
 
+    /**
+     * Updates info about the local member.
+     *
+     * @param member the local member.
+     * @param metadata metadata of the local member.
+     */
+    void updateLocalMember(Member member, @Nullable NodeMetadata metadata) {
```

Comment:
This is strange that we need to pass a `member` to the `updateLocalMember` method. `TopologyService` has a link to the local member already